apps/api-gateway/src/services/prophet_forecast_service.py

```
@@ -0,0 +1,257 @@
+        return self._redis
+
+    def _cache_key(self, store_id: str, metric: str) -> str:
+        return f"prophet:model:{store_id}:{metric}"
+
+    async def _load_model(self, store_id: str, metric: str):
+        """从 Redis 加载缓存的 Prophet 模型"""
+        try:
+            r = await self._get_redis()
+            data = await r.get(self._cache_key(store_id, metric))
+            if data:
+                return pickle.loads(data)
```

Ruff (0.15.2)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)